packages/slice-machine/server/src/api/common/generate.ts

```diff
@@ -0,0 +1,50 @@
+import mock from "@lib/mock/Slice";
+import * as LibrariesState from "./LibrariesState";
+
+import Environment from "@lib/models/common/Environment";
+import { Library, ComponentWithLibStatus } from "@lib/models/common/Library";
+import Files from "@lib/utils/files";
+import { GeneratedPaths } from "@lib/models/paths";
+import { SliceMockConfig } from "@lib/models/common/MockConfig";
+import { getConfig as getGobalMockConfig } from "@lib/mock/misc/fs";
+
+export async function generate(
+  env: Environment,
+  libraries: ReadonlyArray<Library>
+): Promise<void> {
+  try {
+    const components = libraries.reduce<ComponentWithLibStatus[]>(
+      (acc, curr) => [...acc, ...curr.components],
+      []
+    );
+
+    components.forEach(async (c) => {
+      if (
+        !Files.exists(
+          GeneratedPaths(env.cwd)
+            .library(c.from)
+            .slice(c.infos.sliceName)
+            .mocks()
+        )
+      ) {
+        const mocks = await mock(
```

Comment:
I think it could be good to have once and for all a Mock module that does the mock and write in the file.